lib/gems/pending/util/miq-process.rb

```diff
@@ -258,10 +256,3 @@ def self.round_to(number, precision)
     (number * mult).round.to_f / mult
   end
 end
-
-# Examples:
```

Comment:
Do you think these might have a purpose at the top of the file?  Or maybe as documentation for usage of each method?

Reply:
They didn't look very useful to me.